index.html

```diff
@@ -285,6 +285,49 @@ <h5>Skills</h5>
           </li>
         </ul>
       </section>
+      <footer class="contact-me">

```

Comment:
- Hey, let's improve our semantic usage as this should be another section of your webpage and not the footer. kindly change the footer to a section.